docs/ga4gh-pedigree.rst

```diff
@@ -0,0 +1,92 @@
+.. _rsga4ghpedigree:
+
+##############
+GA4GH Pedigree
+##############
+
+
+The Phenopacket-schema includes an implementation of the `GA4GH Pedigree Standard<https://pedigree.readthedocs.io/en/latest/index.html>`_.
+GA4GH Pedigree defines a conceptual model of individuals and their relationships to allow other standards, such as
+Phenopackets to be able to create an implementation of this model using native concepts. For example, the Phenopacket
+schema representation of an individual in the GA4GH Pedigree is a ref:`rstphenopacket`.
+
+The GA4GH Pedigree Standard allows for much greater expressivity of the relationships between individuals compared to the
+PED model used in the ref:`rstfamily` message due to defining the relationships using the `Kinship ontology (KIN) <http://purl.org/ga4gh/kin.owl>`_
```

Comment:
Does the pedigree standard include the KIN ontology? If not, then "due to defining" is wrong and could be replaced by "...message because of the detailed kinship relations defined in the `Kinship ontology ....

Reply:
It's not 'part of' the standard as in being defined directly in that specification, but it is 'required' so it is part of by requirement.